src/deck.jsx

```diff
@@ -275,7 +266,7 @@ class Deck extends React.Component {
   _renderSlide() {
     const slide = this._getSlideIndex();
     const child = this.props.children[slide];
-    return cloneWithProps(child, {
+    return cloneElement(child, {
       key: slide,
```

Comment:
Ok this is a fucking hoot. With 0.14, you have to add `children: React.Children.toArray(child.props.children)` here for it to work. Otherwise, TransitionGroup loses its shit and remounts. I have no idea why.